sanitizer/_size.js

```diff
@@ -1,36 +1,54 @@
 const _ = require('lodash');
-const MIN_SIZE = 1;
-const MAX_SIZE = 40;
-const DEFAULT_SIZE = 10;
+const peliasConfig = require('pelias-config').generate().api;
+//Allow custom min, max and default sizes
+const MIN_SIZE = peliasConfig.minSize || 1;
+const MAX_SIZE = peliasConfig.maxSize || 40;
+let default_size_init;
+if (peliasConfig.DEFAULT_SIZE) {
+  //Cant be smaller/bigger than min/max due to validation in schema.js
+  default_size_init = peliasConfig.DEFAULT_SIZE;
```

Comment:
Even though Joi does some bounds checking I would still do some here, there's no telling if we'll keep using Joi in the future, if it's features will change (we are on a very old version). In either case some logic left in the schema could be lost.

I don't think we need anything fancy with throwing errors, just clamp between MIN_SIZE/MAX_SIZE and if there's an invalid condition leave it at the default of 10.